RetroBar/Controls/NotifyIcon.xaml.cs

```diff
@@ -111,5 +111,33 @@ private void NotifyIcon_OnMouseMove(object sender, MouseEventArgs e)
             e.Handled = true;
             TrayIcon?.IconMouseMove(MouseHelper.GetCursorPositionParam());
         }
+
+        private const int APPCOMMAND_VOLUME_DOWN = 0x90000;
+        private const int APPCOMMAND_VOLUME_UP = 0xA0000;
+        private const string VOLUME_GUID = "7820ae73-23e3-4229-82c1-e41cb67d5b9c";
+
+        private void ChangeVolume(bool upOrDown)
+        {
+            NativeMethods.SendMessage(WindowHelper.FindWindowsTray(IntPtr.Zero), (int)NativeMethods.WM.APPCOMMAND,
+                WindowHelper.FindWindowsTray(IntPtr.Zero), upOrDown ? (IntPtr)APPCOMMAND_VOLUME_UP : (IntPtr)APPCOMMAND_VOLUME_DOWN);
+        }
```

Comment:
Could you move this functionality out from the UserControl into a utility class?